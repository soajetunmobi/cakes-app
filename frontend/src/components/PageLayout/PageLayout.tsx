import React, { FunctionComponent, PropsWithChildren, ReactNode } from 'react'
import { Header } from '../Header'
import { Main } from '../Main'
import { Footer } from '../Footer'

export const PageLayout: FunctionComponent = ({ children }: PropsWithChildren<ReactNode>) => {
  return (
    <>
      <Header />
      <Main>{children}</Main>
      <Footer />
    </>
  )
}
