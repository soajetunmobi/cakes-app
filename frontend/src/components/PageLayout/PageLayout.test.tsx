import React from 'react'
import renderer from 'react-test-renderer'
import { PageLayout } from './PageLayout'

test('it renders correctly', () => {
  const root = renderer.create(<PageLayout />).toJSON()
  expect(root).toMatchSnapshot()
})
