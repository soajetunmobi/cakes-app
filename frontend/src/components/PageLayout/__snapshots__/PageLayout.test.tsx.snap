// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
Array [
  <div
    className="header"
  >
    <h1
      className="logo"
    >
      Cake Reviews
    </h1>
  </div>,
  <main
    className="content"
  />,
  <footer
    className="footer"
  >
    <span>
      Cake Reviews
    </span>
    <br />
    2020
  </footer>,
]
`;
